shifthelper/shepherd.py

```diff
@@ -0,0 +1,53 @@
+import pytz
+import requests
+import dateutil
+import dateutil.parser
+from datetime import datetime
+from twilio.rest import RestClient as Client
+from .tools import config
+import time
+from urllib.parse import urlencode
+
+c = config
+
+url = 'https://ihp-pc41.ethz.ch/time'
+# -> {  "time": "Fri, 07 Apr 2017 10:37:15 GMT" }
```

Comment:
This is just how jsonify({'time': datetime.utcnow()}) behaves ... so its a question of the webinterface ... but I agree it should be ISO 1806 or whatever that was.